hack/.linted_packages

```diff
@@ -388,7 +388,6 @@ staging/src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1alpha1
 staging/src/k8s.io/sample-apiserver
```

Comment:
Do you need to rebase your commit? Looks like this should follow through from where it was added.